packages/config/config/devices/0x010f/fgk101.json

```diff
@@ -303,110 +312,55 @@
 		{
 			"#": "12",
 			"$if": "firmwareVersion >= 3.2",
-			"label": "ON command sent to 2nd assoc. group",
+			"label": "Value of ON Sent to 2nd Association Group",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 255,
 			"defaultValue": 255
 		},
 		{
-			"#": "13",
+			"#": "13[0x01]",
 			"$if": "firmwareVersion <= 3.1",
```

Comment:
I don't think I would split this up tbh. 4 values is simple enough for a single dropdown.